schemamd/render.go

```diff
@@ -157,23 +213,61 @@ func writeBlockChildren(w io.Writer, parents []string, block *tfjson.SchemaBlock
 
 	groups := map[int][]string{}
 
+	// Group Attributes/Blocks by characteristics.
+nameLoop:
```

Comment:
I personally never love named loops... but I see we used to do `goto` so... I suppose this is preferred

Reply:
Exactly

Reply:
Don't love the named loop, and would prefer something like:

```go

for _, n := range names {
  if childBlock, ok := block.NestedBlocks[n]; ok {
    for i, gf := range groupFilters {
      if gf.filter(childBlock) {
        groups[i] = append(groups[i], n)
        break
      }
    }  else if childAtt, ok := block.Attributes[n]; ok {
      // do the same thing
    } else {
      // return error
    }
}
```

or even better

```go
for _, n := range names {
  if childBlock, ok := block.NestedBlocks[n]; ok {
    addToGroup(groups, groupFilters, n)
  } else if childAtt, ok := block.Attributes[n]; ok {
    addToGroup(groups, groupFilters, n)
  }
  // return error
}
```

But refactoring is probably beyond the scope of this PR and we can cover that later.

Reply:
I'm right here people 😭

Reply:
Ahhh! A ghost! 👻